content/docs/CLI/build.md

```diff
@@ -5,7 +5,7 @@ disableToc: false
 weight: 3
 ---
 
-# build
+# `packit build`
```

Comment:
Can you please update this for all the commands? (`validate-config` is still in the old format, for exampl).

Reply:
hmmm, I thought I did, will check again, thanks for noticing!

Reply:
I was indeed naive, many more were not updated, my bad